doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -19,11 +19,17 @@
 
 ## Synopsis
 
-Ensure Reference Implementation (RI) of CNTT Reference Model (RM) and CNTT Reference Architecture (RA) meets industry driven quality assurance standards for compliance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
+Ensure Reference Implementation (RI) of CNTT Reference Model (RM) and CNTT Reference Architecture (RA) meets industry driven quality assurance standards for conformance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
```

Comment:
Is the intention for the testing to only apply to the reference implementation (where there is only 1) or any platform intending to meet the requirements of the RA / RM, specifically commercial distributions. I think it is the latter, and this should be worded as "Ensure an implementation of the CNTT .... (RA), such as the Reference Implementation (RI), meet the ..."

Reply:
The latter.  I reworked this.  Please check to see if resolved.